client/state/transactions/reducer.js

```diff
@@ -0,0 +1,56 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+
+import { combineReducers, createReducer } from 'state/utils';
+import { TRANSACTION_CREATE_REQUEST, TRANSACTION_CREATE_SUCCESS, TRANSACTION_CREATE_FAILURE } from 'state/action-types';
+
+// Gets set and reset on success / failure
+export const isFetching = createReducer(
+	false, // Default request
+	{
+		[ TRANSACTION_CREATE_REQUEST ]: ( state, action ) => action.request,
+		[ TRANSACTION_CREATE_SUCCESS ]: () => false, // reset
+		[ TRANSACTION_CREATE_FAILURE ]: () => false,
+	}
+);
+
+// Gets set and stays set
+export const request = createReducer(
+	false, // Default request
+	{
+		[ TRANSACTION_CREATE_REQUEST ]: ( state, action ) => action.request,
+		[ TRANSACTION_CREATE_SUCCESS ]: ( state, action ) => state, // kept as original action.request
+		[ TRANSACTION_CREATE_FAILURE ]: ( state, action ) => state,
+	}
+);
+
+// Gets set on response
+export const response =	createReducer(
```

Comment:
Can we rename this to something that describes the data rather than its origin as an HTTP response? So, what it is rather than how we got it. Even something like `data` might do, or something you think is appropriate. Just to help others who might be debugging and come across something like `state.transaction.response`.